building/configlet/lint.md

```diff
@@ -418,3 +418,16 @@ The `config.json` file should have the following checks:
 - `%{snake_slug}`: the `snake_case` exercise slug (e.g. `bit_manipulation`)
 - `%{camel_slug}`: the `camelCase` exercise slug (e.g. `bitManipulation`)
 - `%{pascal_slug}`: the `PascalCase` exercise slug (e.g. `BitManipulation`)
+5. Unique version 4 UUID string: A string that satisfies all of these conditions:
+   - It only exists once in the track-level `config.json` file of a given Exercism track
+   - It does not exist in the track-level `config.json` file of any other Exercism track
+   - It does not exist in any `canonical-data.json` file in https://github.com/exercism/problem-specifications
+   - It matches the regular expression:
```

Comment:
The list item style here is inconsistent (see [here](https://github.com/exercism/docs/blob/b07d5b7eb5cc85014fa334756ec787534fc5c2b5/building/configlet/lint.md#glossary)).

Which do we prefer?

Reply:
The one that prettier prefers :) I've just checked and the current version is fine.

Reply:
I've indented the sub-items of item 4. Looks like `prettier` wanted a blank line above and below the previous non-indented version.